inception-project/inception
# Feature/2556 project url slugs

**What's in the PR**
* Ability to specify an URL slug which is used instead of the project ID when constructing URLs
* Also includes some updates to the project settings documentation and updated screenshots

**How to test manually**
* Create a new project, use the pre-defined slug or set one manually
* Use the application (any part) and generally keep an eye on the URL to see if it uses the slug or not

**Automatic testing**

**Documentation**
